src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,47 @@
+package wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Setter
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
```

Comment:
Maybe you can use the lombok Data annotation here

Reply:
I think @Data has a lot of methods that i won't use, i prefer method import than if i will use.